04week/loop.js

```diff
@@ -0,0 +1,52 @@
+// Use a for loop to console.log each item in the array cars
+
+let cars = ['Ford', 'Chevy', 'Honda', 'Toyota'];
+
+for (i = 0; i < cars.length; i ++) {
+  console.log(cars[i]);
+}
+
+/* Use a for...in loop to console.log each key.
+*  Then use a for...in loop and if state to console.log the value associated with the key birthDate
+*/
+
+let persons = {
+  firstName: "Jane",
+  lastName: "Doe",
+  birthDate: "Jan 5, 1925",
+  gender: "female"
+};
+
+for (let prop in persons) {
+  console.log(prop + " is " + persons[prop]);
+}
+
```

Comment:
Missed: Then use a for...in loop and if state to console.log the value associated with the key birthDate.